code/modules/projectiles/guns/projectile/automatic/vintorez.dm

```diff
@@ -25,6 +25,7 @@
 		SEMI_AUTO_NODELAY,
 		FULL_AUTO_400
 		)
+	wield_delay = 3 //deciseconds
```

Comment:
full blown assault rifle , make it 6 deciseconds

Reply:
he went off my table, goes off caliber .20 should be like .4 or .5 iirc

Reply:
done